packages/matchbox/src/components/LabelValue/LabelValue.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { createPropTypes } from '@styled-system/prop-types';
+import styled from 'styled-components';
+import { tokens } from '@sparkpost/design-tokens';
+import { margin } from 'styled-system';
+import { pick } from '../../helpers/systemProps';
+import { Box } from '../Box';
+
+const StyledWrapper = styled.div`
+  ${margin}
+`;
+
+const LabelValue = React.forwardRef(function LabelValue(props, userRef) {
+  const { label, children, className, orientation, ...rest } = props;
+
+  const systemProps = pick(rest, margin.propNames);
+
+  return (
+    <StyledWrapper className={className} {...systemProps} ref={userRef} tabIndex="-1">
+      <Box
+        display={orientation === 'horizontal' ? 'grid' : 'block'}
+        gridGap={orientation === 'horizontal' ? '300' : ''}
+        gridTemplateColumns={orientation === 'horizontal' ? `${tokens.spacing_900} 1fr` : ''}
```

Comment:
Are we always going to want this much horizontal spacing between the label and the value? Do you think we'd want to make this configurable or give it a max width?

Reply:
In the app, the component uses `11rem` which is larger I think, at 176px